include/libtomo/stripe.h

```diff
@@ -53,3 +53,21 @@
 
 DLL void
 remove_stripe_sf(float* data, int dx, int dy, int dz, int size, int istart, int iend);
+
+DLL int
+stripesdetect3d_main_float(float* Input, float* Output, 
+                           int window_halflength_vertical,
+                           int ratio_radius,
+                           int ncores,
+                           long dimX, long dimY, long dimZ);
```

Comment:
Please use `int` for the array shapes. Internally, you can convert to `size_t` if you need to compute the number of elements in the array. The maximum integer on any platform should be sufficient to express any single dimension of any array.